mk/PcapPlusPlus.mk.fuzzing

```diff
@@ -0,0 +1,12 @@
+### FUZZING ###
+
+# includes
+PCAPPP_INCLUDES += -I/usr/include/netinet
+
+# libs
+PCAPPP_LIBS += -lpthread
+
+# allow user to add custom LDFLAGS
+PCAPPP_BUILD_FLAGS += $(LDFLAGS)
+
+COMPILE_FOR_FUZZING := 1
```

Comment:
we can probably remove this variable if we decide not to use it in the main Makefile